docker-hub/billing/upgrade.md

```diff
@@ -77,8 +81,9 @@ To upgrade from a monthly Team plan to an annual Team plan:
 To upgrade from a monthly Pro plan to an annual Pro plan:
 
 1. Log into your [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} account.
-2. Open the drop-down menu next to your username in the top-right corner and select **Billing**. The Billing page displays billing information about your existing user account.
-3. Click **switch to annual billing**.
-4. Review the information displayed on the **Change to an Annual Plan** page and click **Accept Terms and Purchase** to confirm.
+2. Open the drop-down menu next to your username in the top-right corner and select **Billing**. The **Billing Details** page displays billing information about your existing user account.
```

Comment:
It feels abrupt to jump from talking about **Billing** to **Billing Details**; I might not understand the implicit connection - that clicking the former will take you to the latter.

```suggestion
2. Open the drop-down menu next to your username in the top-right corner and select **Billing** to visit the **Billing Details** page, which displays billing information about your existing user account.
```

Reply:
Updated